span-normalizer/span-normalizer-constants/src/main/java/org/hypertrace/core/semantic/convention/constants/db/OTelDbSemanticConventions.java

```diff
@@ -21,7 +23,15 @@
   MONGODB_COLLECTION("db.mongodb.collection"),
   DB_NAME("db.name"),
   REDIS_DB_INDEX("db.redis.database_index"),
-  SQL_TABLE_NAME("db.sql.table");
+  SQL_TABLE_NAME("db.sql.table"),
+  CASSANDRA_DB_NAME("db.cassandra.keyspace"),
```

Comment:
should this be called `CASSANDRA_DB_KEYSPACE`, I understand that it is the equivalent of DB_NAME?

Reply:
yup. It is alternative to DB_NAME for cassandra